{
  return (caught instanceof S3ConnectionException) ? (S3ConnectionException)caught : new S3ConnectionException(message,caught);
}
